# RRT Parameters
step_size: 0.8
obs_threshold: 0.25

xybuffer: 0.0
zbuffer: 2.5
passage_size: 3.5

min_height: 0.9
max_height: 2.5

max_tries: 25
timeout: 1.0
z_scale: 1.0